src/solvers/sat/satcheck_minisat2.cpp

```diff
@@ -29,6 +29,9 @@ Author: Daniel Kroening, kroening@kroening.com
 #error "Expected HAVE_MINISAT2"
 #endif
 
+const double satcheck_minisat2_baset::default_random_seed=91648253.;
```

Comment:
Where does this magic number come from? Explain and move it to util/magic.h.

Reply:
It was copied from minisat2 internals.